deploy/task-cad-checks-secrets-ocm.yaml

```diff
@@ -0,0 +1,11 @@
+---
+# to generate this secret use the command
+# oc create secret generic cad-ocm-token-file -n configuration-anomaly-detection --from-file=${HOME}/.config/ocm/ocm.json  --dry-run -oyaml > task-cad-checks-secrets-ocm.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cad-ocm-token-file
+  namespace: configuration-anomaly-detection
+type: Opaque
+data:
+  ocm.json: dGVzdC1zZWNyZXQ=
```

Comment:
Just a double check: These secrets are okay to be public, because these are only for test purposes, correct?